features/preferences/impl/src/main/res/values/localazy.xml

```diff
@@ -11,6 +11,10 @@
   <string name="screen_advanced_settings_share_presence">"Share presence"</string>
   <string name="screen_advanced_settings_share_presence_description">"If turned off, you won’t be able to send or receive read receipts or typing notifications"</string>
   <string name="screen_advanced_settings_view_source_description">"Enable option to view message source in the timeline."</string>
+  <string name="screen_blocked_users_unblock_alert_action">"Unblock"</string>
+  <string name="screen_blocked_users_unblock_alert_description">"You\'ll be able to see all messages from them again."</string>
```

Comment:
The dialog confirmation not displaying again the user details is not as helpful as it could be. Maybe in this dialog we can render the user name and avatar?

Note: this is how this screen with the dialog renders on the current app:

<img width="409" alt="image" src="https://github.com/element-hq/element-x-android/assets/3940906/2062e5dc-dd58-40d3-bbf5-e92ab1ca87f4">

Reply:
> Maybe in this dialog we can render the user name and avatar?

We don't have that info at this point, and to have them we'd need to modify the Rust SDK. 

> Note: this is how this screen with the dialog renders on the current app:

Adding the user id sounds like a good idea, I'll mention it and see if we change this behaviour, thanks!